proposals/0069-swift-mutability-for-foundation.md

```diff
@@ -122,7 +122,7 @@ let myOtherDate = Date()
 myOtherDate.addTimeInterval(60) // Error, as expected
 ```
 
-It is important to remember that the `Date` API author still controls the methods available on the type, and does not have to provide mutability for every property (as they would in a C struct). For types where we want to provide limited mutability, we can make all properties `get` only and add `mutable` methods to tightly control state and maintain internal consistency. For example, in the case of `Date` the `NSTimeInterval` ivar is still private to the implementation while we provide a `mutable func` to add a time interval.
+It is important to remember that the `Date` API author still controls the methods available on the type, and does not have to provide mutability for every property (as they would in a C struct). For types where we want to provide limited mutability, we can make all properties `get` only and add `mutating` methods to tightly control state and maintain internal consistency. For example, in the case of `Date` the `NSTimeInterval` ivar is still private to the implementation while we provide a `mutating func` to add a time interval.
```

Comment:
When talking about methods, I changed the text to use 'mutating' instead of 'mutable'. I think that makes more sense semantically and the keyword is called 'mutating'.